Makefile

```
@@ -31,6 +31,7 @@ help:
 .PHONY: services
 services: args?=up -d
 services: python
+	@tox -qe dockercompose --run-command 'sh -c "docker network create dbs || true"'
```

```suggestion
	@tox -qe dockercompose --run-command 'sh -c "docker network create dbs 2>/dev/null || true"'
```

Keep failures quiet, as they happen as a matter of course.